.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - name: Set GOLANG_VERSION
+        run: |
+          echo "GOLANG_VERSION=$(go version | awk '{ print $3 }')" >> $GITHUB_ENV
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          registry: docker.io
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
```

Comment:
its strange seeing a hyphen on a blank line

Reply:
This is valid yaml although we should enforce with tooling if preferred to see a specific way